### ISSUE TITLE
Rasterizer

### ISSUE THREAD

**Issue by isaacgerg** [tensorflow/graphics#223]
I would like to replicate figure 3 from the tfg website: https://www.tensorflow.org/graphics/overview

The loss is computed in the image space.  To do this, I believe I need a differentiable renderer so that I can backpropagate the loses to my post estimate network.  However, in looking through the tfg source, I cant find a differentiable rasterizer.  Does one exist?

**Comment by isaacgerg** [comment]
I figured this out, at least for the case of the sphere example.
